### PR TITLE
Jsweek5 and6

### PR DESCRIPTION
**Pull request by lalitchoudhary09**

### CODE REVIEW

**Comment by lalitchoudhary09** [comment]
Exercise 5.1
Exercise 5 1 Screenshot

Exercise 5.2
Exercise 5 2 Screenshot

Exercise 5.3
Exercise 5 3 Screenshot

Exercise 5.4
Exercise 5 4 Screenshot

Problem 6.1
Problem 6 1 sceenshot

Problem 6.2
Problem 6 2 sceenshot

Problem 6.3
Problem 6 3 sceenshot

Problem 6.4
Problem 6 4 sceenshot

Problem 6.5
Problem 6 5 sceenshot

Problem 6.6



@anilrao316

File: Week-6/Problem 6.3.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let low=0;
+let mid=0;
+let high = arr.length-1;
+while(mid<=high)
+{
+    switch(arr[mid])
+    {
+        
+        case 0:
+               arr[low]=arr[mid];
```
Comment (anilrao316) [inline Week-6/Problem 6.3.js:15]: mention time and space complexity for each of the problems

**Comment by anilrao316** [comment]
rest looks good!

File: Week-6/Problem 6.3.js (hunk 1 above)
Comment (lalitchoudhary09) [reply Week-6/Problem 6.3.js:15]: as per you review comment I have added time and space complexity in each assignment.

**Pull request merged**